src/View/Parsers/HTMLValue.php

```diff
@@ -32,7 +32,7 @@ public function __construct($fragment = null)
      */
     public function setContent($content)
     {
-        $content = preg_replace('#</?(html|head|body)[^>]*>#si', '', $content);
+        $content = preg_replace('#</?(html|head(?!er)|body)[^>]*>#si', '', $content);
```

Comment:
Just a comment here — I know it's cooler to use more regex foo, but here it will be much easier for anyone reading the list of tags if it was explicitly listed as `html|head|body|header`.